addons/sale/report/ir_actions_report_templates.xml

```diff
@@ -167,6 +167,10 @@
 
             <div class="oe_structure"/>
 
+            <p t-if="doc.carrier_id.carrier_description" id="carrier_description">
+                <strong>Shipping Description:</strong>
+                <span t-out="doc.carrier_id.carrier_description"/>
+            </p>
```

Comment:
Should be done in delivery, with an inherited template.
(I don't think it can be easily done in a mail template, but a classic qweb template should be easily overridden, and it will be cleaner in terms of module architecture/inheritance)

Reply:
Done. Although imo it looks a bit confusing to have a file called `ir_actions_report_templates.xml` in delivery

Reply:
If you prefer `sale_templates.xml` or another name, you don't have to strictly use the same name than `sale` ^^.

I'm still thinking of a good guideline to split SQL views and pdf reporting, but until a solution is found, in sale, I wanted to avoid `sale_report_templates` or naming like that to avoid confusion with the sale.report model :confused:.